apps/web-giddh/src/app/shared/header/components/group-update/group-update.component.ts

```
@@ -150,11 +155,15 @@ export class GroupUpdateComponent implements OnInit, OnDestroy, AfterViewInit {
 
         this.activeGroup$.subscribe((activeGroup) => {
             if (activeGroup) {
-                this.selectedDiscounts = [];
+                this.selectedDiscounts = null;
                 this.uniqueName = activeGroup.uniqueName;
                 if (activeGroup.applicableDiscounts && activeGroup.applicableDiscounts.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)